src/System.Text.Primitives/System/Text/Encoding/SymbolTable.Symbol.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text
+{
+    public partial class SymbolTable
+    {
+        public enum Symbol : ushort {
+            D0 = (ushort)0,
+            D1 = (ushort)1,
+            D2 = (ushort)2,
+            D3 = (ushort)3,
+            D4 = (ushort)4,
+            D5 = (ushort)5,
+            D6 = (ushort)6,
+            D7 = (ushort)7,
+            D8 = (ushort)8,
+            D9 = (ushort)9,
+            DecimalSeparator = (ushort)10,
+            Exponent = (ushort)16,
+            GroupSeparator = (ushort)11,
+            InfinitySign = (ushort)12,
+            MinusSign = (ushort)13,
+            NaN = (ushort)15,
+            PlusSign = (ushort)14,
```

Comment:
Should we add some common JSON symbols to the table? e.g. '}", '[", etc.? Would it speed up our JSON writer?